release/build_pip_package.sh

```diff
@@ -16,44 +16,66 @@
 set -e
 set -x
 
+# Pick the Python that TFQ/TensorFlow used during configure/build.
+# Order: explicit env -> python3 (>= 3.10)
+PY="${PYTHON_BIN_PATH:-}"
+if [[ -z "${PY}" ]]; then
+  if ! command -v python3 >/dev/null 2>&1; then
+    echo "ERROR: python3 not found. Set PYTHON_BIN_PATH to a Python 3.10+ interpreter." >&2
+    exit 2
+  fi
+
+  # Require Python >= 3.10 for TFQ.
+  if ! python3 - <<'PY'
+import sys
+sys.exit(0 if sys.version_info[:2] >= (3, 10) else 1)
+PY
+  then
+    echo "ERROR: Python 3.10+ required for TensorFlow Quantum; found $(python3 -V 2>&1)." >&2
+    exit 2
+  fi
+
+  PY="$(command -v python3)"
+fi
+echo "Using Python: ${PY}"
+
+# Ensure packaging tools are present in THIS interpreter
+if ! "${PY}" -m pip show -q setuptools wheel >/dev/null 2>&1; then
+  "${PY}" -m pip install --upgrade pip setuptools wheel
+fi
+
 EXPORT_DIR="bazel-bin/release/build_pip_package.runfiles/__main__"
 
-function main() {
-  DEST=${1}
-  EXTRA_FLAGS=${2}
+main() {
+  DEST="$1"
+  EXTRA_FLAGS="$2"
 
-  if [[ -z ${DEST} ]]; then
+  if [[ -z "${DEST}" ]]; then
     echo "No destination directory provided."
     exit 1
   fi
 
-  mkdir -p ${DEST}
+mkdir -p "${DEST}"
```

Comment:
Trivial nit: something happened to the intendentation on this line.

```suggestion
  mkdir -p "${DEST}"
```